docs/deploying-airbyte/on-kubernetes.md

```diff
@@ -43,13 +43,41 @@ Configure `kubectl` to connect to your cluster by using `kubectl use-context my-
 
 ### Configure Logs
 
-Airbyte requires an S3 bucket for logs. Configure this by filling up the following variables in the `.env` file in the `kube/overlays/stable`
-directory:
+Both `dev` and `stable` versions of Airbyte include a stand-alone `Minio` deployment. Airbyte publishes logs to this `Minio` deployment.
+This means Airbyte comes as a **self-contained Kubernetes deployment - no other configuration is required**.
+
+Airbyte currently supports logging to `Minio` or `S3`. The following instructions are for users wishing to log to their own `Minio` layer or `S3` bucket.
+
+#### Configuring Custom Minio Log Location
+Replace the following variables in the `.env` file in the `kube/overlays/stable` directory:
+```text
+# The Minio bucket to write logs in.
+S3_LOG_BUCKET=
```

Comment:
Is the same ENV var used for both Minio and S3 buckets?

Reply:
yes that's right!